bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
```

Comment:
if toml = https://github.com/toml-lang/toml than the language should be toml iirc as the bear will be used on tml. Go is the language the tool you are wrapping is written in. But the language this will analyse is toml.
This is how I understand it